.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "api-enumerations"]
```

Comment:
The Makefile is going to need to be updated to ensure the submodule is present when building the service.
Adding a new update_submodules target to the makefile such as https://github.com/companieshouse/extensions-web/blob/master/Makefile#L67

and calling it as a pre-req for the build target such as: https://github.com/companieshouse/extensions-web/blob/master/Makefile#L21

Should ensure this